cloud/aws/courses/default.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 master:
-  ami: "ami-75bb700d"
-  type: "m4.xlarge"
+  ami: "ami-f573188d"
+  type: "m5.2xlarge"
```

Comment:
@greglarkin m5.**2**xlarge or was this supposed to be just xlarge?

Reply:
@marrero984 Decided to go with `2xlarge` because David D (?) reported slowdowns in the PDX classroom a couple of weeks ago with 18 students. The cost differential isn't too much since the instance is only around for a few days and the m5 instances are a little cheaper than the m4 instances. There may be some extra capacity that goes unused for smaller classes, but perhaps overall performance will be better anyway.